fms/modules/positions.py

```diff
@@ -234,7 +234,7 @@ def adjusted_qk(
         assert len(q.size()) == 4
```

Comment:
btw the docs above for q/k are outdated:

`Embedded query tensor, expected size is B x H x S x Eh`